addons/website/models/website.py

```diff
@@ -342,7 +342,7 @@ def configurator_init(self):
 
     @api.model
     def configurator_recommended_themes(self, industry_id, palette):
-        domain = [('name', '=like', 'theme%'), ('name', 'not in', ['theme_default', 'theme_common'])]
+        domain = [('name', '=like', 'theme%'), ('name', 'not in', ['theme_default', 'theme_common']), ('state', '!=', 'uninstallable')]
         client_themes = request.env['ir.module.module'].search(domain).mapped('name')
         client_themes_img = dict([(t, http.addons_manifest[t].get('images_preview_theme', {})) for t in client_themes])
```

Comment:
Out of curiosity, did you find this by just reading the code and trying to find a possible issue or did you really found a community theme which was set as uninstallable in the manifest?

I am not sure why would someone do that

Reply:
@rdeodoo That's the real situation for me. My git has a branch of 15.0 copied from 14.0 for upgrade, all modules that have not been upgraded I set installable=False

Reply:
That's why I got this error.

Reply:
Thanks for your suggestion! I was hesitant to add this condition at first :joy:
Could you check again, please?